lib/reversetunnel/srv.go

```diff
@@ -1107,7 +1112,7 @@ func newRemoteSite(srv *server, domainName string, sconn ssh.Conn) (*remoteSite,
 		return nil, err
 	}
 
-	go remoteSite.updateCertAuthorities(caRetry)
+	go remoteSite.updateCertAuthorities(caRetry, remoteClusterVersion)
```

Comment:
nit: instead of forwarding `remoteClusterVersion` from two layers it can be just saved as a member in `remoteSite`  struct.

Reply:
I was thinking about it, the `remoteSite struct` is already quite huge and looks like the version is not needed anywhere. Let me look at this once again.